docs/products/access/topics/look-and-feel/templates/templates.md

```diff
@@ -1,42 +1,45 @@
+OneWelcome Access uses the [Thymeleaf](https://www.thymeleaf.org/) template engine to serve pages to the end user via a browser. The templates can be customized to
 align their look and feel with your mobile application or company standards.
 
 ## Customizing templates
 
-Onegini Access uses template sets which contain one or more custom templates. The default template is used when a specific template is not available in the
+Access uses template sets that contain one or more custom templates. The default template is used when a specific template is not available in the
 template set. One custom template set can be marked as the default template set for all applications and web clients.
 
-Customized templates can be uploaded and managed via the [Self Styling application](../../../../../self-styling). In order to add the new template set follow
+Customized templates can be uploaded and managed via the [Self Styling application](../../../../../self-styling). To add the new template set, follow
 the [directory structure](../../../../../self-styling/configuration-page.md)
 in the configured git repository. Refer to the [default template set](./content/access-engine-templates.zip) to further customize templates.
 
-In the templates the [Thymeleaf Layout Dialect](https://github.com/ultraq/thymeleaf-layout-dialect) and
-[Thymeleaf With Dialect](https://github.com/Antibrumm/thymeleaf-extras-with-dialect) are supported. The template file names are hardcoded and should be equal in
-the custom template set. The internals of the template like message keys and fragments can be different. For more details about the different templates
-supported by Onegini Access see the [templates section in the appendix](../../../appendix/templates/templates.md).
+In the templates, the [Thymeleaf Layout Dialect](https://github.com/ultraq/thymeleaf-layout-dialect) and
+[Thymeleaf With Dialect](https://github.com/Antibrumm/thymeleaf-extras-with-dialect) are supported. The template file names are hardcoded and should not be changed in
+the custom template set. The internals of the template, like message keys and fragments, can be different. For more details about the different templates
+supported by Access see the [templates section in the appendix](../../../appendix/templates/templates.md).
 
-## Configuring an application specific template set
+## Configuring an application-specific template set
 
-For each mobile application and web client a template set can be configured to render screens in the look and feel of this application or web client. When no
-template set is configured for an application or web client, the default template set is used to render these screens.
+For each mobile application and web client, a template set can be configured to render screens in the look and feel of this application or web client. When no
+template set is configured for an application or web client. The default template set is used to render these screens.
```

Comment:
`When no template set is configured for an application or web client. The default template set is used to render these screens.`
shouldn't it be one sentence?